backend/src/main/java/de/neuefische/CapStone/backend/schedulingTask/ScheduleService.java

```diff
@@ -1,28 +1,38 @@
 package de.neuefische.CapStone.backend.schedulingTask;
 
+import de.neuefische.CapStone.backend.service.OpenHabService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.scheduling.config.ScheduledTask;
 import org.springframework.stereotype.Service;
 
 @Service
 public class ScheduleService implements Runnable{
```

Comment:
das hier ist eine Datei die nahezu identisch ist zu einer anderen die bereits da ist. Ich will beim speichern meiner Kaffeemaschine in die Datenbank gleich auch das Einschalt und AusschaltEvent starten. Wenn ich die kurz nacheinander aufrufe, schreiben die sich aus irgendeinem Grund über... das hier ist die hässliche Workaround Lösung, brauche da eher Hilfe von den Coaches